### DISCUSSION TITLE
GitHub Support Non-responsive for 3 weeks

### DISCUSSION THREAD

**Discussion by chinesebug** [community/community#60542] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

General

### Feature Area

Issues

### Body

We have an issue open for 3 weeks now and have had no response. We are applying for non-profit pricing as 501c3. We are currently paying to continue our development at an unsustainable price. Please help before we reach the next bill.

Ticket 2214701

### Guidelines

- [X] I have read the above statement and can confirm my post is relevant to the GitHub feature areas [Issues](https://docs.github.com/en/issues/tracking-your-work-with-issues/about-issues) and/or [Projects](https://docs.github.com/en/issues/planning-and-tracking-with-projects/learning-about-projects/about-projects).

**Comment by GhostOf0days** [comment]
Could you please retry creating a support ticket with Github [here](https://support.github.com/)?

**Comment by chinesebug** [comment]
Hi Jasial,

I have created a new support ticket 

Best,
Matt

**Reply by GhostOf0days** [reply]
Happy to help. Could you please mark this question as closed?

**Reply by GhostOf0days** [reply]
Also, please edit that comment to remove the support ticket number for your privacy/security. Thank you.

**Reply by chinesebug** [reply]
Hi @GhostOf0days , Thanks for the response. Could I ask how long would it take to process the request?

**Reply by GhostOf0days** [reply]
Usually, Github Support responds in 1-3 days.

**Reply by queenofcorgis** [reply]
Hi @chinesebug, 

Thank you for your patience. The community is unable to assist with nor escalate your ticket, but we can assure you it is in the right place. It will be answered in the order it was received based on our team's resources and the amount of tickets ahead of yours. We appreciate your understanding! In the future, please only open 1 ticket per incident.

As no other Community Discussions team members will be able to provide additional help with account related questions I am going to close this discussion. Please note: the Issues category is solely related to conversations around the GitHub products Issues and Projects.

Thank you!
